sass/_variables.scss

```diff
@@ -154,7 +154,7 @@ $mobile_portrait_length: 480px !default;
 
     --occupants-padding: 1em;
     --occupants-background-color: white;
-    --occupants-border-left: 3px solid var(--controlbox-pane-border-color);
+    --occupants-border-left: 3px solid var(--panel-divider-color);
```

Comment:
Hi @bernard-ng, sorry, could you please make one more change.

We should try avoiding absolute measurements such as pixels, and instead use relative ones like `em` or `rem`.

Can you please set `3px` to how many `rem` is necessary to have the same effect?